climada/hazard/tc_tracks.py

```diff
@@ -953,6 +953,23 @@ def from_simulations_storm(cls, path, years=None):
         -------
         tracks : TCTracks
             TCTracks with data from the STORM simulations.
+
+        Notes
+        -----
+        All tracks are set in the year 1980. The id of the year (starting from 0) is saved in the
+        attribute 'id_no'. To obtain the year of each track use
+        
+        >>> years = [int(tr.attrs['id_no'] / 1000) for tr in tc_tracks.data]
+        >>> # or, alternatively, 
+        >>> years = [int(tr.attrs['sid'].split("-")[-2]) for tr in tc_tracks.data]
+
+        If a windfield is generated from these tracks using the method `TropCylcone.from_tracks()`,
+        the following should be considered:
+        1. The frequencies will be set to ``1`` for each storm. Thus, in order to compute annual
+        values, the frequencies of the TropCylone should be changed to ``1/number of years``.
+        2. The storm year and the storm id are stored in the ``TropCyclone.event_name`` attribute.
+
+
```

Comment:
I played around with the formatting in https://livesphinx.herokuapp.com/ and found that the spacing has to be a bit different, actually:
```suggestion
        If a windfield is generated from these tracks using the method ``TropCylcone.from_tracks()``,
        the following should be considered:

        1. The frequencies will be set to ``1`` for each storm. Thus, in order to compute annual
           values, the frequencies of the TropCylone should be changed to ``1/number of years``.
        2. The storm year and the storm id are stored in the ``TropCyclone.event_name`` attribute.
```